website/blog/2023-11-30-sharing-podman-images-with-kubernetes-cluster.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Share your local podman images with the Kubernetes cluster
+description: Build image and use it directly in your kubernetes cluster
+slug: sharing-podman-images-with-kubernetes-cluster
+authors: [benoitf]
+tags: [podman-desktop, podman, images, kubernetes]
+hide_table_of_contents: false
+image: /blog/img/sharing-podman-images-with-kubernetes-cluster/selkie-family.png
+---
+
+As developers we constantly improve and refine our applications. One of the challenges we face is quickly iterating when working with container images and kubernetes deployments/pods.
+
+For example, when we want to try a new image in a kubernetes pod, this image needs to be available on a private/public registry or on the nodes of the kubernetes cluster.
+Sometimes we need to call extra commands such as `kind load docker-image` or `minikube cache add <image>` or publish the image first to a 3rd party registry.
+
+You'll agree that trying out a new image in a Kubernetes pod should be as seamless as building the image itself.
+
+In this blog post, we will explore the best practice for streamlining the image iteration process in Kubernetes with Podman Desktop.
+
+![hero](img/sharing-podman-images-with-kubernetes-cluster/selkie-family.png)
+
+<!--truncate-->
+
+---
+
+## Introduction
+
+When using kind or minikube or other 3rd party tools to setup a local kubernetes cluster, we have several ways to publish images.
+
+Minikube published 8 ways of doing that at https://minikube.sigs.k8s.io/docs/handbook/pushing/
+
+There are pros and cons either way. Starting a registry requires publishing your image each time you want to build. Podman Desktop could automate the synchronization but there is still a duplication of layers.
```

Comment:
I have a doubt about this sentence. 
'Starting a registry requires publishing your image each time you want to build.'
What are you trying to say? It makes me think that a registry (like quay.io) only start if i publish an image and i publish my image when i want to build it.
But i guess the meaning is more "by starting a registry you are allowed to publish your image each time you build it"? 🤔

Reply:
the idea was that if you're using a 3rd party registry, you need to publish the image to it, registry being local or remote is not changing the cons

Reply:
ok, not sure how to rewrite this tbh. Maybe someone else has a better idea or it's good this way 👍